.github/CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 Contributing
 ------------------
 
-Thank you for considering contributing to [Ensono](https://www.ensonodigital.com) Stacks. It's people like you that continue to make Stacks the best it can be.
+Thank you for considering contributing to [Ensono Stacks](https://www.ensonodigital.com). It's people like you that continue to make Ensono Stacks the best it can be.
```

Comment:
Should this URL be [stacks.ensono.com/docs/contributions](https://stacks.ensono.com/docs/contributions)?